libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -817,6 +830,16 @@ void AP_Vehicle::check_motor_noise()
 #endif
 }
 
+#if AP_DDS_ENABLED
+[[nodiscard]] bool AP_Vehicle::init_xrce_client()
+{
+    if (AP::serialmanager().have_serial(AP_SerialManager::SerialProtocol_DDS_XRCE, 0)) {
+        xrce_client = new AP_DDS_Client();
```

Comment:
variable should be dds_client, not xrce_client

Reply:
Done in 361686e332dd36232b7c313185d2669f2d6af3dd